modules/metering-troubleshooting.adoc

```diff
@@ -10,10 +10,173 @@ A common issue with metering is pods failing to start. Pods might fail to start
+A common issue when installing or running metering is a lack of compute resources. As the cluster grows and more reports are created, the `reporting-operator` pod requires more memory. If memory usage reaches the pod limit, the cluster terminates the pod with an `OOMKilled` status. Ensure that metering is allocated the minimum resource requirements described in the installation prerequisites.
+
+NOTE: CPU usage for the `reporting-operator` pod does not increase as the cluster grows.
 
 To determine if the issue is with resources or scheduling, follow the troubleshooting instructions included in the Kubernetes document https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#troubleshooting[Managing Compute Resources for Containers].
 
+To troubleshoot issues due to a lack of compute resources, check the following within the `operator-metering` namespace.
+
+.Procedure
+
+. Check for metering `Report` resources that fail to complete and show the status of `ReportingPeriodUnmetDependencies`:
++
+[source,terminal]
+----
+$ oc get reports
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                  QUERY                          SCHEDULE   RUNNING                            FAILED   LAST REPORT TIME       AGE
+namespace-cpu-utilization-adhoc-10    namespace-cpu-utilization                 Finished                                    2020-10-31T00:00:00Z   2m38s
+namespace-cpu-utilization-adhoc-11    namespace-cpu-utilization                 ReportingPeriodUnmetDependencies                                   2m23s
+namespace-memory-utilization-202010   namespace-memory-utilization              ReportingPeriodUnmetDependencies                                   26s
+namespace-memory-utilization-202011   namespace-memory-utilization              ReportingPeriodUnmetDependencies                                   14s
+----
+
+. Check the `ReportDataSource` resources where the `NEWEST METRIC` is less than the report end date:
++
+[source,terminal]
+----
+$ oc get reportdatasource
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                         EARLIEST METRIC        NEWEST METRIC          IMPORT START           IMPORT END             LAST IMPORT TIME       AGE
+...
+node-allocatable-cpu-cores                   2020-04-23T09:14:00Z   2020-08-31T10:07:00Z   2020-04-23T09:14:00Z   2020-10-15T17:13:00Z   2020-12-09T12:45:10Z   230d
+node-allocatable-memory-bytes                2020-04-23T09:14:00Z   2020-08-30T05:19:00Z   2020-04-23T09:14:00Z   2020-10-14T08:01:00Z   2020-12-09T12:45:12Z   230d
+...
+pod-usage-memory-bytes                       2020-04-23T09:14:00Z   2020-08-24T20:25:00Z   2020-04-23T09:14:00Z   2020-10-09T23:31:00Z   2020-12-09T12:45:12Z   230d
+----
+
+. Check the health of the `reporting-operator` pod for a high number of pod restarts:
++
+[source,terminal]
+----
+$ oc get pods
```

Comment:
This is also sufficient if it's just a mirror of that KCS article, but we can also just filter out that reporting-operator controller Pod, by running `oc get pods -l app=reporting-operator` and the output would be a single Pod that matches that label.